frida_tools/tracer.py

```diff
@@ -514,11 +542,13 @@ def _create_resolver_script(self):
 
 
 class Tracer(object):
-    def __init__(self, reactor, repository, profile, log_handler=None):
+    def __init__(self, reactor, repository, profile, init_session, log_handler=None):
         self._reactor = reactor
         self._repository = repository
         self._profile = profile
         self._script = None
+        self._init_session_paths = init_session
+        self._init_session_code = ''
```

Comment:
It appears this variable is only used locally below, so there's no need to keep it around right?

Reply:
Good catch!  I have removed `self._init_session_code`, making it local to `_create_trace_script()`:

```
    def _create_trace_script(self):
        var init_session_code = ''
        for path in self._init_session_paths:
            with open(path, 'rt') as fo
                lines = fo.readlines()

            lines = [line.rstrip() for line in lines]

            # eval() does not like 0x0A, replace with '\u000a'
            init_session_code += '\\u000a'.join(lines)
            
            # Insert '\ua000' in case another init session follows
            init_session_code += '\\u000a'

        return """'use strict';
...
""" % {"init_code": init_session_code}
```

Reply:
Cool! Remember to use double-quotes since it's considered a string and not an enum value.

Reply:
OK, done.